src/entities/communitySbt.ts

```diff
@@ -855,7 +837,7 @@ class SBT {
       this.currentBadgesSubgraphUrl,
       this.nextBadgesSubgraphUrl,
     );
-    const subgraphClaimedBadges = await this.claimedBadgesInSubgraph(
+    const subgraphClaimedBadges = await SBT.claimedBadgesInSubgraph(
```

Comment:
again, why SBT and not this?

Reply:
because the function is ``static`` at this point